src/cpu_shader/euler.rs

```diff
@@ -204,10 +227,10 @@ impl EulerSeg {
         )
     }
 
+    // Note: offset provided is scaled so that 1 = chord length
```

Comment:
Consider renaming `offset` to `normalized_offset` or `scaled_offset` or something along those lines. I think it doesn't hurt to make this obvious, now that the scaling is not done internally.

Same for `EulerParams::eval_with_offset` which had the same expectation to begin with.